core/src/main/resources/jenkins/model/Jenkins/login.properties

```diff
@@ -20,5 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-signUp=<a href="signup">Create an account</a> if you are not a member yet.
+signUp=Please sign in below or <a href="signup">create an account</a>.
```

Comment:
Potentially should wipe existing translations.

Reply:
Not sure what you mean.

Reply:
If there are `login_*.properties` translated files, wipe out the new obsolete translations.

Reply:
well, yeah this is how jenkins work, right?

Reply:
I am asking you to check `login_*.properties` files for existing translations of this key, and remove them, as they are now probably obsolete and need to be translated again.

Reply:
Actually I think not since I only changed the caption of the english translation but the translations of it are not false (only may missing the "sign in below" part but IMO that is not enough to remove the key from the translations.